conf/test_contamination.config

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
This test profile is almost identical to test_full_filter_contamination isn't it?